src/ecobalyse_data/food/README.md

```diff
@@ -0,0 +1,75 @@
+# Utiliser la librairie python "Brightway2"
+
+La librairie python [brightway2](https://brightway.dev/) est un framework
+d'analyse du cycle de vie. Elle est très pratique pour pouvoir importer des
+données dans une base locale, faire des requêtes et calculs, et pour ensuite
+pouvoir exporter des données.
+
+Dans le répertoire [brightway/import_agb/](import_agb/) vous trouverez
+un script permettant d'importer la base de données
+[agribalyse](https://agribalyse.ademe.fr/), et dans le répertoire
+[brightway/export_agb/](export_agb/) le script utilisé pour exporter
+des données sous format json.
+
+Pour utiliser ces scripts, il vous faudra installer le gestionnaire de paquets
+[anaconda](https://docs.conda.io/projects/conda/en/latest/) et créer un
+environnement virtuel.
+
+Dans le répertoire
+[brightway/brightway_tutorial_notebooks/](brightway_tutorial_notebooks/)
+vous trouverez plusieurs exemples d'utilisation de la librairie brightway2 sous
+forme de [notebooks jupyter](https://jupyter.org/) pour explorer et expérimenter
+par vous-même.
+
+## Installer anaconda et créer son environnement virtuel
+
+Suivre la [procédure d'installation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+Créer un environnement virtuel avec toutes les dépendences requises :
+
+    $ conda env create -f food/environment.yml
```

Comment:
Argh, now I see that instructions were there too. THat should probably be stated in the README at the root of the repo⋅⋅⋅